utilities/cam_test.py

```diff
@@ -354,10 +354,10 @@ def exit_cleanly(signum, frame):
     EXP_STEP = 500  # us
     ISO_STEP = 50
     LENS_STEP = 3
-    DOT_STEP = 100
-    FLOOD_STEP = 100
-    DOT_MAX = 1200
-    FLOOD_MAX = 1500
+    DOT_STEP = 1 / 12
+    FLOOD_STEP = 1 / 15
```

Comment:
Why are we stepping by different factors? Would be clearer (for the user) if both steps are just 0.1

Reply:
The idea was to have the same amount of steps as when stepping through the current @ 100mA. But that's really it, would be simpler to use the same step, maybe make it smaller since there is a bit of non-linearity going from 0.1 to i think ~0.2 (ends up being quite a jump in intensity).

Reply:
Lets normalize for "intensity" in FW already (intensity = k*current^0.5 or something along those lines) and later on we can introduce a normalized power [0,1], where we'd control it in the existing manner